lib/SILOptimizer/Mandatory/TFPartition.cpp

```diff
@@ -17,6 +17,7 @@
 
 #define DEBUG_TYPE "tf-partition"
 #include "TFUtilities.h"
+#include "tensorflow/c/c_api.h"
```

Comment:
Gate with `#ifdef SWIFT_ENABLE_TENSORFLOW`? This is done in [TFLowerGraph.cpp](https://github.com/apple/swift/blob/tensorflow/lib/SILOptimizer/Mandatory/TFLowerGraph.cpp#L20).

Reply:
+1

Reply:
Done